lib/stripe/subscriptions/subscription.ex

```diff
@@ -108,10 +108,13 @@ defmodule Stripe.Subscription do
                optional(:collection_method) => String.t(),
                optional(:coupon) => Stripe.id() | Stripe.Coupon.t(),
                optional(:days_until_due) => non_neg_integer,
-               optional(:items) => [
+               :items => [
```

Comment:
Should there be a pattern match on the presence of `:items` in the `Subscription.create/2` call just like it's done for `:customer`?
 (and possible a guard on whether the value passed to `:items` is  a list).

Reply:
Oh great idea!